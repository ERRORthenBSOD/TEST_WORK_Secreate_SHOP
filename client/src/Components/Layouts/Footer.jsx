import React from 'react';
import {AppBar, Toolbar, Typography,} from '@material-ui/core';
import {makeStyles} from '@material-ui/core/styles';

const useStyles = makeStyles(theme => ({
	text: {
		padding: theme.spacing(2, 2, 0),
	},
	paper: {
		paddingBottom: 50,
	},
	list: {
		marginBottom: theme.spacing(2),
	},
	subheader: {
		backgroundColor: theme.palette.background.paper,
	},
	appBar: {
		top: 'auto',
		bottom: 0,
		zIndex: theme.zIndex.drawer + 1,
	},
	grow: {
		flexGrow: 1,
	},
	fabButton: {
		position: 'absolute',
		zIndex: 1,
		top: -30,
		left: 0,
		right: 0,
		margin: '0 auto',
	},
	fontWeight: {
		fontWeight: '600'
	},
	favColor:{
		color: theme.palette.favorite,
	}
}));


const Footer = () => {
	const classes = useStyles();
	return (
		<AppBar position="fixed" color="primary" className={classes.appBar}>
			<Toolbar>
				<div className={classes.grow} />
				<Typography variant="h6"  align="center" className={classes.fontWeight}>
					«Магазин» - тестовое задание для компании Secreate.io
				</Typography>
			</Toolbar>
		</AppBar>
	);
};


export default Footer;
