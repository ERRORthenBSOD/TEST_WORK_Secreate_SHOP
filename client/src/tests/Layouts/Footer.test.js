import React from 'react';
import { shallow } from 'enzyme';
import Footer from '../../Components/Layouts/Footer'

it('Footer renders without crashing', () => {
    const wrapper = shallow(<Footer />);
    expect(wrapper).toMatchSnapshot()
});