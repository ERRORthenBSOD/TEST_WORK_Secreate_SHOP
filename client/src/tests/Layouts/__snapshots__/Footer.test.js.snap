// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders without crashing 1`] = `
<WithStyles(ForwardRef(AppBar))
  className="makeStyles-appBar-5"
  color="primary"
  position="fixed"
>
  <WithStyles(ForwardRef(Toolbar))>
    <div
      className="makeStyles-grow-6"
    />
    <WithStyles(ForwardRef(Typography))
      align="center"
      className="makeStyles-fontWeight-8"
      variant="h6"
    >
      «Магазин» - тестовое задание для компании Secreate.io
    </WithStyles(ForwardRef(Typography))>
  </WithStyles(ForwardRef(Toolbar))>
</WithStyles(ForwardRef(AppBar))>
`;
